dnf-behave-tests/createrepo_c/steps/lib/file.py

```diff
@@ -35,6 +35,8 @@ def get_compression_suffix(type_str):
         return ".xz"
     if type_str == "bz2":
         return ".bz2"
+    if type_str == "zck":
+        return ".zck"
```

Comment:
Hmm, the `type_str == "zck"` check is already present on line 32 :slightly_smiling_face: 

Seems you could just do:
```python
if type_str in ("gz", "zck", "xz", "bz2"):
    return "." + type_str
# ...
```